DataFormats/Detectors/TPC/src/CalibdEdxCorrection.cxx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DataFormatsTPC/CalibdEdxCorrection.h"
+
+#include <algorithm>
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <string_view>
+
+// o2 includes
+#include "DataFormatsTPC/Defs.h"
+
+// root includes
+#include "TFile.h"
+
+using namespace o2::tpc;
+
+size_t CalibdEdxCorrection::stackIndex(const StackID& stack, ChargeType charge)
```

Comment:
Perhaps you can put the calculation directly in the return statement and put it directly in the header.